LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
Should add a note to the license in the readme file and update the changelog (see https://github.com/ONSdigital/sdx-collect/pull/170/files)